src/color-system.mdx

```diff
@@ -0,0 +1,98 @@
+
+As you notice, for each hue there is a `-default` and `-light` swatch available in this palette. The `-light` tints are usually good to use when creating interactive visual feedback. For example a button that has `--color-green-default` as background color, can get `--color-green-light` when it is hovered by the cursor.
+___
+
+#### **Primary palette**
+Includes 40 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+<limel-example-primary-color-palette />
+
+Sometimes, just a handful of colors is not enough and you need a bit more. For example when you need colors for your icons, or diagrams. This palette is designed to provide a decent number of harmonic colors while easing the choice of colors for you. Used for primary purposes and meaningful communications with colors.
+
+Note that this palette also includes brand colors, and contrast swatches.
+___
+
+#### **Extended palette**
+Includes 100 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+
+<limel-example-extended-color-palette />
+
+This palette offers a much richer set of colors that can be used for visual communications, iconography, illustrations, and charts.
+
+___
+## Lime Technologies' Brand Colors
+<limel-example-brand-color-palette />
+
+These set of colors are included in all available palettes, and can be used for branding and marketing purposes. Swatches and their respective CSS variables have specific names starting with the prefix of `--lime-brand-color-`. All of these hues already exists in our color palette and mostly reside in the "dark mode" subset -as stated beside their names.
+
+**Note** that names of our brand colors are chosen by our Marketing department thus sometimes do not exactly match our naming conventions for their equivalent hues in the color pallets.
+
+**Note** that the brand colors do **not** have a dark and light version, and when their respective CSS variable names are used, they will be displayed with the same brightness in both light and dark UIs.
+
+### Usage of brand colors
+For correct use of Lime's brand assets, including colors, please refer to [Lime Technologies' brand guidelines](https://www.lime-technologies.se/brand-guidelines/).
+
+We recommend you to rely on the "UI palette" and avoid using brand colors extensively in UI design. Brand colors -as their name states- should be used where appropriate, to reflect the company's brand & identity; and add brand-recognizability to the product. These should be used like the way you use spices in cooking. Too much and too many of them together will destroy the food. Therefore in UI design, usage of brand colors should be very limited.
```

Comment:
Perhaps all discussion of brand colors vs palette colors and their difference with respect to dark-mode should be moved here instead?